config/jobs/periodic/docker/periodic-ci-docker.yaml

```diff
@@ -0,0 +1,207 @@
+            exit $rc
+          securityContext:
+            privileged: true
+    - name: periodic-integration-cli-docker
+      cluster: k8s-ppc64le-cluster
+      labels:
+        preset-build-docker: "true"
+      cron: 0 14 * * *
+      decorate: true
+      decoration_config:
+        timeout: 3h
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:b89afc315f96f5cfd43de82b3b02f9221983592b893f838d816db5954088c9fc
+          resources:
+            requests:
+              cpu: "2000m"
+            limits:
+              cpu: "2000m"
+          command:
+          - /bin/bash
+          args:
+          - -c
+          - |
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            set -o xtrace
+            echo "* Start prow-integration-tests *"
+            chmod ug+x $PWD/upstream-master-ci/prow-integration-cli.sh
+            $PWD/upstream-master-ci/prow-integration-cli.sh
+            rc=$?
+            [ $rc != 0 ] && echo "ERROR: prow-integration-tests exited with code:$rc"
+            exit $rc
+          securityContext:
+            privileged: true
+    - name: periodic-integration-test-docker
+      cluster: k8s-ppc64le-cluster
+      labels:
+        preset-build-docker: "true"
+      cron: 30 14 * * *
+      decorate: true
+      decoration_config:
+        timeout: 3h
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:b89afc315f96f5cfd43de82b3b02f9221983592b893f838d816db5954088c9fc
+          resources:
+            requests:
+              cpu: "4000m"
+            limits:
+              cpu: "4000m"
+          command:
+          - /bin/bash
+          args:
+          - -c
+          - |
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            set -o xtrace
+            echo "* Start prow-integration-tests *"
+            chmod ug+x $PWD/upstream-master-ci/prow-integration-tests.sh
+            $PWD/upstream-master-ci/prow-integration-tests.sh
+            rc=$?
+            [ $rc != 0 ] && echo "ERROR: prow-integration-tests exited with code:$rc"
+            exit $rc
+          securityContext:
+            privileged: true
```

Comment:
The [lint job failure](https://prow.ppc64le-cloud.cis.ibm.net/view/s3/ppc64le-prow-logs/pr-logs/pull/ppc64le-cloud_test-infra/517/pull-test-infra-yamllint/1866778956489822208) indicates that I'm missing a new line (I think the hyphen indicates that I don't have a blank line at the end of the file since it doesn't have a line number before it). Should I add a newline?

Reply:
Yes try adding a new line. I sometimes encounter EOF-related lint errors when editing YAML files using inappropriate editors.
To avoid such issues, it’s best to use a proper YAML-friendly editor that ensures consistent formatting. 

If needed, you can always rely on the pull-test-infra-yamllint job to catch and fix YAML formatting issues. For example, refer to this configuration:
[pull-test-infra-yamllint Job Configuration](https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/test-infra/test-infra-presubmits.yaml#L3:L15).